deployments/master.yml

```
@@ -179,8 +186,10 @@ Mappings:
       Version: ${{PANTHER_VERSION}} # E.g. "1.14.0"
 
 Conditions:
+  FullDeployment: !Equals [!Ref FullDeployment, 'true']
   RegistryProvided: !Not [!Equals [!Ref ImageRegistry, '']]
   OnboardSelf: !Equals [!Ref OnboardSelf, true]
+  CreateOnboard: !And [!Condition FullDeployment, !Condition OnboardSelf]
```

I don't think you need the `OnboardSelf` condition anymore, this could just be

```suggestion
  CreateOnboard: !And [!Condition FullDeployment, !Equals [!Ref OnboardSelf, true]]
```

(with the logic flipped if we rename the parameter as I suggest above)